src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignment_submissions/test_lmsassignment_submissions.py

```diff
@@ -0,0 +1,337 @@
+            LMSAssignmentSubmission = query(
+                connection, "SELECT * from [lmsx].[AssignmentSubmission]"
+            )
+
+            assert len(LMSAssignmentSubmission) == 1
+            assert (
+                int(LMSAssignmentSubmission[0]["AssignmentSubmissionIdentifier"]) == SUBMISSION_TEST_LMS_IDENTIFIER
+            )
+
+
+def describe_when_there_are_assignment_submissions_to_update():
+    SIS_SECTION_ID = "sis_section_id"
+    ASSIGNMENT_SOURCE_SYSTEM_IDENTIFIER = "assignment_identifier"
+    ASSIGNMENT_CATEGORY = "test_category"
+    ASSIGNMENT_SUBMISSION_STATUS = "test_submission_status"
+    USER_SIS_ID = "test_sis_id"
+    SUBMISSION_TEST_IDENTIFIER = "submission_test_identifier"
+    SUBMISSION_TEST_LMS_IDENTIFIER = 99
+
+    def it_should_update_the_submissions_successfully(test_db_config: ServerConfig):
+        # arrange
+        with MSSqlConnection(test_db_config).pyodbc_conn() as connection:
+
+            insert_descriptor(connection, DESCRIPTOR_NAMESPACE, ASSIGNMENT_CATEGORY)
+            insert_lmsx_assignmentcategory_descriptor(connection, 1)
+
+            insert_descriptor(connection, DESCRIPTOR_NAMESPACE, SOURCE_SYSTEM)
+            insert_lmsx_sourcesystem_descriptor(connection, 2)
+
+            insert_descriptor(
+                connection,
+                SUBMISSION_STATUS_DESCRIPTOR_NAMESPACE,
+                ASSIGNMENT_SUBMISSION_STATUS,
+            )
+            insert_lmsx_assignmentsubmissionstatus_descriptor(connection, 3)
+
+            insert_lms_section(connection, SIS_SECTION_ID, SOURCE_SYSTEM)
+            insert_edfi_section(connection, SIS_SECTION_ID)
+            connection.execute(
+                """UPDATE LMS.LMSSECTION SET
+                    EdFiSectionId = (SELECT TOP 1 ID FROM EDFI.SECTION)"""
+            )
+
+            insert_lms_assignment(
+                connection,
+                ASSIGNMENT_SOURCE_SYSTEM_IDENTIFIER,
+                SOURCE_SYSTEM,
+                1,
+                ASSIGNMENT_CATEGORY,
+            )
+
+            insert_lms_user(connection, USER_SIS_ID, USER_TEST_EMAIL, SOURCE_SYSTEM)
+            insert_edfi_student(connection, USER_SIS_ID)
+            connection.execute(
+                """UPDATE LMS.LMSUSER SET
+                    EdFiStudentId = (SELECT TOP 1 ID FROM EDFI.Student)"""
+            )
+
+            insert_lms_assignment_submissions(
+                connection,
+                SUBMISSION_TEST_LMS_IDENTIFIER,
+                SUBMISSION_TEST_IDENTIFIER,
+                1,
+                1,
+                ASSIGNMENT_SUBMISSION_STATUS,
+                SOURCE_SYSTEM,
+                False,
+            )
+
+        run_harmonizer(test_db_config)
+        with MSSqlConnection(test_db_config).pyodbc_conn() as connection:
+            connection.execute(
+                """
+                UPDATE LMS.ASSIGNMENTSUBMISSION SET
+                    GRADE=N'99',
```

Comment:
Make grade value a constant.  Also not same as SUBMISSION_TEST_LMS_IDENTIFIER!

Reply:
👍🏼